app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -72,9 +72,9 @@ public static SettingsFragment newInstance(Bundle intentArgs, SettingsScreen set
             case REMOVE_ENGINES:
                 f = new SettingsFragment();
                 break;
-            case ADD_SEARCH:
+            /*case ADD_SEARCH:
```

Comment:
When removing code, you can just delete it – no need to comment it out! If we need to revive the code, we can get it from version control.

That being said, I'm not convinced we should delete this – see below.